### ISSUE TITLE
Unit Copying Issues 

### ISSUE THREAD

**Issue by sarahlazarich** [metrotranscom/doorway#658]
Babs has called out some issues she's run into when copying units during listing creation. 

When drafting listings that charge rent based on income: 
- When I indicate the percentage of income (so far it's always been 30), and select Save & Exit or select Make a Copy, and then I go back into the unit to edit the details, the percentage of income drops by one, and sometimes three (aka, 29, 27). I can edit the percentage to 30 and it will save. 
- Similar to how the % of income decreases from 30 to 29, etc. when copies are made, the max income for 8 people decreases as well, as copies are made. 

AC: 
- Unit copies should be the same. The fields should not increase or decrease as you make copies.

**Comment by jaredcwhite** [comment]
@sarahlazarich I spent some time on this, and I don't think I could reproduce it. I _might_ have noticed it one time, but I might have been mistaken. I hope it's not one of those things that's very intermittent?

**Comment by emilyjablonski** [comment]
We chatted about the first issue, it's something that happens to me sometimes when I'm using a trackpad or a mouse, if you're focused into an input element it's native to the browser that you can scroll up and down to change the value, and sometimes it can get stuck in there

**Comment by jaredcwhite** [comment]
@emilyjablonski Yeah, it might be a funky aspect of how we're using `type="number"` input fields, and most of the instances I found in a codebase search are contained in that particular form. I wonder if it's best to add an event handler to disable mouse wheel changing numbers (possible solution here: https://stackoverflow.com/a/69497807), or instead switch to regular text input and use other attributes to force integer input, a number keyboard on mobile, etc. (looks like we could go with `inputmode="numeric" pattern="[0-9]*" type="text"` as mentioned here: https://css-tricks.com/finger-friendly-numerical-inputs-with-inputmode/#aa-a-better-solution-inputmode)

**Comment by jaredcwhite** [comment]
OK, so we'll fix this first in Core via https://github.com/bloom-housing/bloom/pull/4128, then bring the patch over.
